shotover/src/transforms/kafka/sink_cluster.rs

```diff
@@ -54,6 +56,9 @@ pub struct KafkaSinkClusterBuilder {
     shotover_nodes: Vec<KafkaAddress>,
     connect_timeout: Duration,
     read_timeout: Option<Duration>,
+    coordinator_broker_id: Arc<AtomicI32>,
+    topics: Arc<DashMap<StrBytes, Topic>>,
```

Comment:
In CassandraSinkCluster we just use a RwLock<HashMap> for concurrent access to a hashmap, did you find the performance benefit from this new dependency DashMap to be worth it?

Reply:
Well using a concurrent hashmap instead of a RwLock *should* be faster, but I cant measure anything till we get windsock kafka benchmarks to be stable enough.
Definitely something to experiment with once we get better benchmarks.